### PR TITLE
Prl 4846 Manage order - Judge approval journey- Approval scenarios

### PR DESCRIPTION
**Pull request by BharadwajSairam**
JIRA link (if applicable)

https://tools.hmcts.net/jira/browse/PRL-4846

Change description

Manage order - Judge approval journey- Approval scenarios

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/private-law/json/AuthorisationCaseField/EditAndApprove/AuthorisationCaseField.json [hunk 1]
```diff
@@ -677,5 +677,154 @@
...
+            "caseworker-privatelaw-courtadmin",
+            "caseworker-privatelaw-systemupdate",
+            "caseworker-privatelaw-superuser",
+            "caseworker-privatelaw-la",
+            "caseworker-privatelaw-judge"
+          ],
+          "CRUD": "CRU"
+        },
+        {
+          "UserRoles": [
```
Comment (shashisk97) [inline definitions/private-law/json/AuthorisationCaseField/EditAndApprove/AuthorisationCaseField.json:735]: Any reason for not using idam roles? if we are doing it here then it has to be consistent accross solicitor journey

File: definitions/private-law/json/CaseEventToFields/EditAndApprove/Page4/CaseEventToFields.json [hunk 2]
```diff
@@ -9,88 +9,143 @@
...
+  {
+    "LiveFrom": "31/03/2022",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "editAndApproveAnOrder",
+    "CaseFieldID": "orderNameEditScreenLabel",
+    "PageFieldDisplayOrder": 4,
+    "DisplayContext": "READONLY",
+    "PageID": 4,
+    "PageDisplayOrder": 4,
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/judge-or-admin-populate-draft-order-common-fields",
```
Comment (shashisk97) [inline definitions/private-law/json/CaseEventToFields/EditAndApprove/Page4/CaseEventToFields.json:49]: Is this callback needed? If yes can this be move to first field in the page?

File: definitions/private-law/json/FixedLists.json [hunk 3]
```diff
@@ -6539,5 +6539,54 @@
...
+    "LiveFrom": "05/12/2023",
+    "ID": "JudgeApprovalDecisionsCourtAdminEnum",
+    "ListElementCode": "editTheOrderAndServe",
+    "ListElement": "Edit the order myself and send to admin to serve",
+    "DisplayOrder": 3
+  },
+  {
+    "LiveFrom": "05/12/2023",
+    "ID": "JudgeApprovalDecisionsSolicitorEnum",
+    "ListElementCode": "sendToAdminToServe",
```
Comment (shashisk97) [inline definitions/private-law/json/FixedLists.json:6567]: These element codes needs to be changed for Admin and solicitor, both have the same element code.

Comment (BharadwajSairam) [reply definitions/private-law/json/FixedLists.json:6567]: As long as Id is unique we can make use of same ListelementCodes

**Review by yogendraU — APPROVED** [review]

**Comment by BharadwajSairam** [comment]
Created 1775, closing this since no longer required

**Pull request closed without merging**